src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -3,6 +3,71 @@
 <html>
 	<head>
 		<script>
+			async function gitpodMetricsAddCounter(metricsName, labels, value) {
+				function getMetricsUrl() {
+					const baseWorkspaceIDRegex = '(([a-f][0-9a-f]{7}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})|([0-9a-z]{2,16}-[0-9a-z]{2,16}-[0-9a-z]{8,11}))';
```

Comment:
@mustard-mh @iQQBot as far as I remeber we use inline replacement while serving workbench.html, maybe it would be easier to compute metrics endpoint during the build and let blobserve to replace with generated value while serving. It would prevent us spreading the code. nothing to do now, just an idea